src/gui/qgsdoublevalidator.h

```diff
@@ -0,0 +1,155 @@
+#ifndef QGSDOUBLEVALIDATOR_H
```

Comment:
convention is copyright at immediate top of file, followed by header guard

Reply:
Done !